src/scene/vcFlythrough.cpp

```diff
@@ -254,6 +256,8 @@ void vcFlythrough::HandleSceneEmbeddedUI(vcState *pProgramState)
 
   if (m_state != vcFTS_Exporting)
   {
+    vcIGSW_FilePicker(pProgramState, vcString::Get("flythroughExportPath"), m_exportPath, udLengthOf(m_exportPath), nullptr, 0, vcFDT_SelectDirectory, [] {});
```

Comment:
Pass in `nullptr` instead of an empty lambda, this is how this is used elsewhere.